README.md

```diff
@@ -60,11 +60,3 @@ This section has moved here: [https://facebook.github.io/create-react-app/docs/m
 ### Advanced Configuration
 
 This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
```

Comment:
Please add back this section

Reply:
Done [Revert deployment section](https://github.com/abed-961/todo-list-react/pull/1/commits/74802ac8e90ae37570c27cb8773e0d97b90c89c7)